filebeat/tests/system/test_harvester.py

```diff
@@ -66,8 +66,10 @@ def test_close_renamed(self):
             lambda: self.output_has(lines=iterations1 + 1), max_timeout=10)
 
         # Wait until registry file is created
+        # Because the default flush timeout is 1s, we might have only one message
```

Comment:
Can you modify the test configuration to use 0s? Unless we are explicitly trying to test the flush interval I would rather not having every test that inspects the registry file take an additional 1s.

Reply:
My idea is to have those integration tests reflecting that most users will use (aka default configuration), in the end of  the day, they're different code paths.

Based on the tests that broke when I implemented the change, they don't seem to be that many, so I believe the increase in time to run all tests isn't very significative.

Does it make sense to you?

Reply:
As long as the increase is test time isn't noticeable we don't need to change anything. In the ideal world we should have one test that the delay works and every other test should be configured to execute as fast as possible.

Reply:
Actually hang on, why are we waiting 60s? 60s is forever in a test case. Can the timeout be shortened here?

These things have a way of multiplying via copy pasting test cases over time (like months) and then we start wondering why CI is so slow :)

Reply:
Is normal execution shorter than 60s?

Reply:
I don't know why, but up until now I was reading this `max_timeout=65` like `max_timeout` was in a `1/60s` unit :rofl: :man_facepalming::man_facepalming::man_facepalming::man_facepalming:

Thanks for catching this!


65s is definitely way too much to wait. I updated them all to be `2s` and did some quick comparison of the test files I modified:
- My branch: `47 passed in 72.11s (0:01:12)`
- `main`: `47 passed in 59.74s`

It seems reasonable to keep the default flush time.